### PR TITLE
Feature/teaching page evals

### PR DESCRIPTION
**Pull request by klpittman**
Adds a new row to the teaching page course cards for course evals. Easiest to test with user "bill".

### CODE REVIEW

File: myuw/static/js/ws_data.js [hunk 1]
```diff
@@ -224,6 +224,7 @@ WSData = {
             var minutes_till_deadline = grading_deadline.diff(ref, 'minutes');
             var deadline_in_24_hours = (minutes_till_deadline >= 0 &&
                                         minutes_till_deadline <= (24 * 60));
+            var comparison_date = moment(new Date(window.card_display_dates.comparison_date));
```
Comment (vegitron) [inline myuw/static/js/ws_data.js]: I think for these moment calls you should have moment parse the string rather than Date.

Comment (klpittman) [inline myuw/static/js/ws_data.js]: @mikeseibel I think these might be more in your department :)

Comment (mikeseibel) [inline myuw/static/js/ws_data.js]: turns out  that was masking a gradepage client bug that's returning string "None" for empty dates in the json payload...

File: myuw/templates/handlebars/card/instructor_schedule/course_eval.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                            <li class="evaluation-sechedule">Report Available {{evaluation.report_available_date_display}}</li>
+                        {{/if}}
+                        {{#if evaluation.report_url}}
+                          <li><a href="{{evaluation.report_url}}">View Report</a></li>
+                        {{/if}}
+                        <li class=""><strong>{{evaluation.response_rate_percent}}% Response rate</strong></li>
+                    {{/if}}
+
+                  {{else}}<!-- not online, paper based -->
+                      <li class="evaluation-sechedule">Paper-based evaluation</li>
```
Comment (bexar) [inline myuw/templates/handlebars/card/instructor_schedule/course_eval.html]: Is "sechedule" a misspelling of schedule? (also added on other list items above and below)

File: myuw/templates/handlebars/card/instructor_schedule/course_eval.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+                      {{/if}}
+                    {{else}} <!-- no evaluation date, not scheduled -->
+                        {{#if evaluation.report_available_date}}
+                            <li class="evaluation-sechedule">Report Available {{evaluation.report_available_date_display}}</li>
+                        {{/if}}
+                        {{#if evaluation.report_url}}
+                          <li><a href="{{evaluation.report_url}}">View Report</a></li>
+                        {{/if}}
+                        <li class=""><strong>{{evaluation.response_rate_percent}}% Response rate</strong></li>
```
Comment (bexar) [inline myuw/templates/handlebars/card/instructor_schedule/course_eval.html]: Remove empty class?

**Comment by bexar** [comment]
External links should have the data-linklabel attribute.

**Comment by klpittman** [comment]
myuw__teaching
Just noticed a separator has gone missing. trying to fix now.

**Comment by charlon** [comment]
There was too much difficulties in trying to target the last element in a "myuw-card-section" class. The solution was to explicitly use the "last" class. Inside of handlebars each loops, we can use the @last selector and add the css class that way (see schedule.html).

**Pull request merged**